### PR TITLE
Freeze gRPC versions installed via pip

### PR DESCRIPTION
**Pull request by mtopolnik**
Add description here

Checklist
- [x] Tags Set
- [x] Milestone Set

### CODE REVIEW

File: extensions/python/src/main/resources/jet_to_python_init.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -e -x
 
 python3 -m venv jet_to_python_env --system-site-packages
 source jet_to_python_env/bin/activate
-python3 -m pip install protobuf grpcio
+python3 -m pip install protobuf==3.11.3 grpcio==1.26.0
```
Comment (cangencer) [inline extensions/python/src/main/resources/jet_to_python_init.sh]: I think it should be possible to use maven properties here and add a resource transformation step

Comment (cangencer) [reply extensions/python/src/main/resources/jet_to_python_init.sh]: see for example `jet-runtime.properties`

Comment (cangencer) [reply extensions/python/src/main/resources/jet_to_python_init.sh]: I see for example in python module we use proto 3.9 and 3.11 here and also different grpc versions

Comment (mtopolnik) [reply extensions/python/src/main/resources/jet_to_python_init.sh]: That's updated as well.

**Pull request merged**